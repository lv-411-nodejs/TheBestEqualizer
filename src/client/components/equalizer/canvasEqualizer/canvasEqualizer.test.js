import React from 'react';
import Enzyme, { shallow, render, mount } from 'enzyme';
import Adapter from  'enzyme-adapter-react-16';
import  CanvasEqualizer  from './canvasEqualizer';

Enzyme.configure({ adapter: new Adapter})

describe('Grafic equaliser', () => {

    const props = {
        width: 150,
        height: 100,
        getCanvasEl: ()=>{},
    }

    it('should RENDER PROPERLY', () => {
        const nextProps = {
            ...props,
        }

        const nextContainer = shallow(<CanvasEqualizer {...nextProps}/>);

    expect(nextContainer).toMatchSnapshot();    

    })
});


