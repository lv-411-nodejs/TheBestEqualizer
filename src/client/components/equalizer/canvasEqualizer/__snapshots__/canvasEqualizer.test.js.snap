// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grafic equaliser should RENDER PROPERLY 1`] = `ShallowWrapper {}`;
